zh/enable-tls-for-dm.md

```diff
@@ -589,6 +590,15 @@ spec:
         ssl-ca: /var/lib/source-tls/${tidb_secret_name}/ca.crt
         ssl-cert: /var/lib/source-tls/${tidb_secret_name}/tls.crt
         ssl-key: /var/lib/source-tls/${tidb_secret_name}/tls.key
+
+    mysql-instances:
+      -
```

Comment:
Should combine this with the next line?

Reply:
https://github.com/pingcap/docs-tidb-operator/pull/1218/commits/d8810e9364e6c112c64c7ec60b3471bbdcf4e59e